### DISCUSSION TITLE
field's conditions (hide -> unhide)

### DISCUSSION THREAD

**Discussion by iqraaaswad** [directus/directus#8152] [Archive]
i have field/button's condition like this 

![image](https://user-images.githubusercontent.com/58161890/133961133-9237d691-7e82-4958-bd3c-22635560e86f.png)

i have set default value for that field (button) to "Hidden on Detail"
![image](https://user-images.githubusercontent.com/58161890/133962304-ee73cd05-0343-4b30-859b-abec72361876.png)

when "mystatus" field changed to 2 (and then saved), that button still invisible for me when i opened that item

**Comment by joselcvarela** [comment]
I think you are confusing `Hidden on detail`

To achieve what you want, you need to disable `Hidden on detail` inside `Fields`, and enable `Hidden on Detail` inside `Conditions`

**Reply by iqraaaswad** [reply]
ah.. i forgot to use **"2"**.. btw thanks for your support
